src/monitoringSetup.js

```diff
@@ -173,8 +172,7 @@ function setupStatuspage(params) {
   spCmd += spNames.map((name) => ` --name "${name}"`).join('');
   if (params.spGroup) spCmd += ` --group "${params.spGroup}"`;
   if (params.incubator) spCmd += ' --incubator true';
-  console.info(`Executing: ${spCmd}`);
-  shell.exec(spCmd, (code, stdout) => {
+  shell.exec(spCmd, { silent: true }, (code, stdout) => {
```

Comment:
I had to silence output from statuspage, because it otherwise appeared in front of the expected test JSON output